src/preferences/dialog/dlgprefkey.cpp

```diff
@@ -128,8 +128,8 @@ void DlgPrefKey::loadSettings() {
         }
     }
 
-    setNotation(notation_type);
     KeyUtils::setNotation(notation);
```

Comment:
What was the reason for swapping the invocations? Then maybe add a comment that explains why the order is relevant.

Reply:
`KeyUtils::setNotation(notation)` stores the (custom) notation map,
and in `setNotation()` the QLineEdits are populated from KeyUtils.
the wrong order was one reason the initial population was done from a default map.
Comment added, squashed, rebased

Reply:
Thank you. This is another example why code with hidden side effects must strictly be avoided.